app/src/main/java/br/com/cantinho/android_mvvm_clean_architecture/ui/calculator/usecases/AddictionUseCase.kt

```diff
@@ -3,9 +3,9 @@ package br.com.cantinho.android_mvvm_clean_architecture.ui.calculator.usecases
 import br.com.cantinho.android_mvvm_clean_architecture.ui.calculator.OperationResult
 import br.com.cantinho.android_mvvm_clean_architecture.ui.calculator.Success
 
-class SumUseCase {
+class AddictionUseCase {
 
-    fun performSum(paramA: Int, paramB: Int): OperationResult {
+    fun performAddiction(paramA: Int, paramB: Int): OperationResult {
```

Comment:
I think the user case name is already meaningfully. Maybe it's better change method's name `performAddiction` to `perform` only.

Reply:
resolved